stdlib/@tests/stubtest_allowlists/common.txt

```diff
@@ -111,6 +111,7 @@ tkinter.simpledialog.TclVersion
 tkinter.simpledialog.TkVersion
 tkinter.Misc.after  # we intentionally don't allow everything that "works" at runtime
 traceback.TracebackException.from_exception  # explicitly expanding arguments going into TracebackException __init__
+types.MappingProxyType.get
```

Comment:
dict is under the "should fix" category, so putting it here as well. I think we should revisit both ignores at some point.